HabitTracker.AshtonLeeSeloka/DataService/DataServices.cs

```diff
@@ -0,0 +1,312 @@
+using Microsoft.Data.Sqlite;
+using System.Globalization;
+using DBItem;
+namespace DataService
+{
+	public class DataServices
+	{
+		public List<DBItems> dataRecords = new List<DBItems>();
+
+		/// <summary>
+		/// General methode used to recieve user input
+		/// </summary>
+		/// <param name="messageToUser">Message to write to console</param>
+		/// <returns>USer input as a string value</returns>
+		public string UserUnput(string messageToUser) 
+		{
+			Console.WriteLine(messageToUser);
+			return Console.ReadLine(); 
+		}
+
+		/// <summary>
+		/// Validates user date input and saves to variable if succesful
+		/// </summary>
+		/// <param name="messageToUser">Prompt displayed to user</param>
+		/// <returns>Correct date value or null</returns>
+		public string? DateInput(string messageToUser) 
+		{
+			Console.Write(messageToUser);
+			string? date = Console.ReadLine();
+
+			if (date == "0")
+				return null;
+
+			while (!DateTime.TryParseExact(date, "dd-MM-yyyy", new CultureInfo("en-Us"), DateTimeStyles.None, out _)) 
```

Comment:
🟠 **Inconsistent Format**

💡 What is it, `dd-MM-yyyy` (line 34) or `dd-MM-yy` (line 36)?

Reply:
I have corrected the format to "dd-MM-yyyy" for all instance.
Thanks :)